### PR TITLE
Add fuma wrapper

### PR DESCRIPTION
**Pull request by shiltemann**
Please sign off by admins only
------------------------------

* [x] Has no duplicate (otherwise merge required)
* [x] Indentation is correct
* [x] Tests
  - [x] Parameters are reasonably covered
  - [x] Test files are appropriate
* [x] `<command/>`: file names are escaped
* [x] Tool requires version/build increase

### CODE REVIEW

File: packages/package_fuma_2_10_0/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools..
+name: package_fuma_2_10_0
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_fuma_2_10_0/.shed.yml:23]: update & give admin access

File: packages/package_fuma_2_10_1/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools..
+name: package_fuma_2_10_1
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_fuma_2_10_1/.shed.yml:23]: update and give access

File: packages/package_fuma_2_11_0/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools..
+name: package_fuma_2_11_0
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_fuma_2_11_0/.shed.yml:23]: update and give access

File: packages/package_fuma_2_11_3/.shed.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools.
+name: package_fuma_2_11_3
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_fuma_2_11_3/.shed.yml:23]: update and give access

File: packages/package_fuma_2_7_1/.shed.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools..
+name: package_fuma_2_7_1
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_fuma_2_7_1/.shed.yml:23]: update and give access

File: packages/package_fuma_3_0_0/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools.
+name: package_fuma_3_0_0
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline packages/package_fuma_3_0_0/.shed.yml:23]: update and give access

File: tools/fuma/.shed.yml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  fusion genes based on gene-name annotations. FuMa automatically
+  compares and summarizes all combinations of two or more datasets in a
+  single run, without additional programming necessary. FuMa uses only
+  one single gene annotation, avoiding mismatches caused by tool
+  specific gene annotations. Furthermore, FuMa accepts intermediate
+  output files that allow a step wise analysis of corresponding tools.
+name: fuma
+owner: yhoogstrate
+homepage_url: https://github.com/yhoogstrate/fuma
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/fuma_galaxy_wrapper
```
Comment (shiltemann) [inline tools/fuma/.shed.yml:27]: update and give access

File: tools/fuma/fuma.xml [hunk 8]
```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tool id="fuma" name="FuMa" version="3.0.0-g0">
+    <description>match detected fusion genes based on gene names (in particular for RNA-Seq)</description>
+    
+    <requirements>
+        <requirement type="package" version="2.7">python</requirement>
+        <requirement type="package" version="3.0.0">fuma</requirement>
```
Comment (yhoogstrate) [inline tools/fuma/fuma.xml]: 3.0.3 & re-test  & add conda-dep

**Pull request merged**